app/src/main/java/org/wikipedia/page/PageActivity.java

```diff
@@ -703,9 +703,11 @@ public void onActivityResult(int requestCode, int resultCode, final Intent data)
                 && resultCode == RESULT_OK) {
             pageFragment.refreshPage();
             String editLanguage = StringUtils.defaultString(pageFragment.getLeadImageEditLang(), app.language().getAppLanguageCode());
-            FeedbackUtil.makeSnackbar(this, TextUtils.isEmpty(pageFragment.getLeadImageEditLang()) ? getString(R.string.description_edit_success_saved_image_caption_snackbar)
-                    : getString(R.string.description_edit_success_saved_image_caption_in_lang_snackbar, app.language().getAppLanguageLocalizedName(editLanguage)), FeedbackUtil.LENGTH_DEFAULT)
-                    .setAction(R.string.suggested_edits_article_cta_snackbar_action, v -> pageFragment.openImageInGallery(editLanguage)).show();
+            if (!Prefs.shouldShowSuggestedEditsSurvey()) {
```

Comment:
I think we should not hide the Snackbar here since the survey Snackbar will only show on the Explore feed.